external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaContinuousTest.scala

```diff
@@ -45,11 +45,15 @@ trait KafkaContinuousTest extends KafkaSourceTest {
     testUtils.addPartitions(topic, newCount)
     eventually(timeout(streamingTimeout)) {
       assert(
-        query.lastExecution.executedPlan.collectFirst {
-          case scan: ContinuousScanExec
-              if scan.stream.isInstanceOf[KafkaContinuousStream] =>
-            scan.stream.asInstanceOf[KafkaContinuousStream]
-        }.exists(_.knownPartitions.size == newCount),
+        query.lastExecution.executedPlan.exists {
```

Comment:
ditto

Reply:
[0031b96](https://github.com/apache/spark/pull/35833/commits/0031b96ae460fa0adf42ae14d0ea6c66b5df0501) revert this changed.